server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -454,4 +455,28 @@ public void deleteMailsTasksShouldCompleteWhenSenderIsValid() {
                 .body("status", is("completed"));
     }
 
+    @Test
+    public void reprocessingAllTaskShouldCreateATask() throws Exception {
```

Comment:
does it test that the task has been serialized?

Reply:
yes, but anyway I will improve the tests to be more coherent